service/deliveryService.ts

```diff
@@ -0,0 +1,13 @@
+import { deliveryEmployee } from "../model/deliveryEmployee";
+
+const axios = require('axios');
+
+module.exports.createDeliveryEmployee = async function(deliveryEmployee: deliveryEmployee): Promise<number>{
```

Comment:
If you could export this the same way as the Controller you will be able to get rid of the require statement where you import this as well